test/support/live_views/elements.ex

```diff
@@ -153,6 +153,9 @@ defmodule Phoenix.LiveViewTest.ElementsLive do
     <form id="trigger-form-value" action="/not_found" method="POST" phx-submit="form-submit-trigger"
           phx-trigger-action={@trigger_action}>
     </form>
+
+    <%# @page_title assign is unique %>
+    <svg><title>SVG with title</title></svg>
```

Comment:
This addition breaks [an existing test for `page_title/1`](https://github.com/phoenixframework/phoenix_live_view/blob/a5fec1d110de8638d1a5042df33e66429b7e2b7a/test/phoenix_live_view/integrations/layout_test.exs#L77). This seems to like such an edge case as to not justify the overhead of a distinct test case.